champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/axe/WindBurst.java

```diff
@@ -0,0 +1,218 @@
+    private void windBurst(Player player, int level) {
+        if (getLevel(player) <= 0) return;
+
+        Location location = player.getLocation().add(0, 1, 0);
+        double maxRadius = getRadius(level);
+
+        List<LivingEntity> enemies = UtilEntity.getNearbyEnemies(player, location, getRadius(level));
+        Vector direction2 = player.getEyeLocation().getDirection();
+
+        VelocityData selfVelocityData = new VelocityData(direction2, selfVelocity, false, ySetSelf, yAddSelf, yMaxSelf, false);
+        UtilVelocity.velocity(player, player, selfVelocityData, VelocityType.CUSTOM);
+        taskScheduler.addTask(new BPVPTask(player.getUniqueId(), uuid -> !UtilBlock.isGrounded(uuid), uuid -> {
+            Player target = Bukkit.getPlayer(uuid);
+            if(target != null) {
+                championsManager.getEffects().addEffect(player, player, EffectTypes.NO_FALL,getName(), (int) fallDamageLimit,
+                        50L, true, true, UtilBlock::isGrounded);
+            }
+        }, 1000));
+
+        for (LivingEntity enemy : enemies) {
+            if (!hitEntities.containsKey(enemy)) {
+                Double yTranslate = location.add(0, -1, 0).getY();
+                Location enemyLocation = enemy.getLocation();
+                enemyLocation.setY(yTranslate);
+                Vector direction = enemyLocation.toVector().subtract(location.toVector()).normalize();
+                VelocityData enemyVelocityData = new VelocityData(direction, velocity, false, ySet, yAdd, yMax, true);
+                UtilVelocity.velocity(enemy, player, enemyVelocityData, VelocityType.CUSTOM);
+                UtilDamage.doCustomDamage(new CustomDamageEvent(enemy, player, null, EntityDamageEvent.DamageCause.CUSTOM, getDamage(level), false, "Wind Burst"));
+
+                hitEntities.put(enemy, true);
+
+            }
+        }
+
+        new BukkitRunnable() {
+            double currentRadius = 0;
+            int ticks = 0;
+
+            @Override
+            public void run() {
+                if (ticks > burstDuration) {
+                    this.cancel();
+                    return;
+                }
+                currentRadius = (maxRadius / burstDuration) * ticks;
+
+                spawnParticles(location, currentRadius);
+                ticks++;
+            }
+        }.runTaskTimer(this.champions, 0, 1);
+
+        player.getWorld().playSound(player.getLocation(), Sound.ENTITY_BREEZE_WIND_BURST, 1.0f, 1.0f);
+    }
+
+
+
+    private void spawnParticles(Location center, double radius) {
+        int numParticles = 10;
+
+        for (int i = 0; i < numParticles; i++) {
+            double theta = Math.random() * 2 * Math.PI;
+            double phi = Math.acos(2 * Math.random() - 1);
+            double x = center.getX() + (radius * Math.sin(phi) * Math.cos(theta));
+            double y = center.getY() + (radius * Math.sin(phi) * Math.sin(theta));
+            double z = center.getZ() + (radius * Math.cos(phi));
```

Comment:
Reuse `Math.sin(phi)`, `Math.cos(phi)`, `Math.cos(theta)`, `Math.sin(theta)` instead of computing them multiple times. Reduces overhead